src/index.html

```diff
@@ -2,13 +2,52 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="li">
+            <a class="nav_link is-active" href="#">Apple</a>
+          </li>
+          <li class="li">
+            <a class="nav_link" href="#">Samsung</a>
+          </li>
```

Comment:
```suggestion
          <li class="li">
            <a class="nav_link is-active" href="#">Apple</a>
          </li>

          <li class="li">
            <a class="nav_link" href="#">Samsung</a>
          </li>
```

Add blank lines between multiple sibling blocks